sopel/db.py

```diff
@@ -367,6 +387,25 @@ def set_channel_value(self, channel, key, value):
         finally:
             session.close()
 
+    def delete_channel_value(self, channel, key):
+        """Sets the value for a given key to be associated with the channel."""
```

Comment:
`delete_channel_value` "Sets" something?

Reply:
I will fix that later tonight, it wouldnt be a dbb PR without a docstring issue